lib/rspec/core/formatters/console_codes.rb

```diff
@@ -7,14 +7,22 @@ module ConsoleCodes
         # @private
         VT100_CODES =
           {
-            :black   => 30,
-            :red     => 31,
-            :green   => 32,
-            :yellow  => 33,
-            :blue    => 34,
-            :magenta => 35,
-            :cyan    => 36,
-            :white   => 37,
+            :black         => 30,
+            :red           => 31,
+            :green         => 32,
+            :yellow        => 33,
+            :blue          => 34,
+            :magenta       => 35,
+            :cyan          => 36,
+            :white         => 37,
+            :brightblack   => '1;30',
+            :brightred     => '1;31',
+            :brightgreen   => '1;32',
+            :brightyellow  => '1;33',
+            :brightblue    => '1;34',
+            :brightmagenta => '1;35',
+            :brightcyan    => '1;36',
+            :brightwhite   => '1;37',
             :bold    => 1,
```

Comment:
If you're aligning the above, this should also be aligned.

Reply:
Personally I'm not a fan of aligning because of the extra and unnecessary lines of diff, but I was just following what was. I fixed the :bold alignment and the rest of your code comments.